XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Extensions/SystemNumericsExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace XRTK.WindowsMixedReality.Extensions
+{
+    public static class SystemNumericsExtensions
```

Comment:
This should be in core as it's using a fairly common type that may not just only be used in the wmr package.